api/src/pcapi/routes/backoffice/templates/accounts/update_requests_list.html

```diff
@@ -74,12 +74,22 @@ <h2 class="fw-light">
                                    data-bs-target="#accept-{{ update_request.dsApplicationId }}">Accepter le dossier</a>
                               </li>
                             {% endif %}
-                          </ul>
-                        </div>
+                          {% endif %}
+                          {% if update_request.can_be_corrected %}
+                            <li class="dropdown-item p-0">
+                              <a class="btn btn-sm d-block w-100 text-start px-3"
+                                 data-bs-toggle="modal"
+                                 data-bs-target="#ask-for-correction-{{ update_request.dsApplicationId }}">Faire une demande de correction</a>
```

Comment:
Je lis dans le ticket : « Via le meatball menu sur la ligne du dossier, ajouter une action “Photo Illisible” »
```suggestion
                                 data-bs-target="#ask-for-correction-{{ update_request.dsApplicationId }}">Photo illisible</a>
```